plugins/samson_ledger/test/initializers/ledger_test.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+require_relative "../test_helper"
+
+SingleCov.covered! file: 'plugins/samson_ledger/config/initializers/samson_ledger/client.rb'
+
+describe 'Ledger::Client' do
+  before do
+    stub_request(:post, "https://foo.bar/api/v1/events")
+  end
+  after do
+    ENV.delete("LEDGER_BASE_URL")
+    ENV.delete("LEDGER_TOKEN")
+  end
+
+  describe "plugin enabled" do
+    around do |test|
+      ENV.delete("LEDGER_BASE_URL")
```

Comment:
using an around as a before ?

... the after outside should already take care of that